runtime/bcverify/vrfyhelp.c

```diff
@@ -931,10 +940,15 @@ isFieldAccessCompatible(J9BytecodeVerificationData *verifyData, J9ROMFieldRef *f
 				 * the field is accessible only when it exists in the subclass or the superclass's
 				 * initialization is completed in which case uninitializedThis is set to FALSE.
 				 */
-				J9BranchTargetStack *liveStack = (J9BranchTargetStack *)verifyData->liveStack;
-				return findFieldFromCurrentRomClass(romClass, fieldRef) || !liveStack->uninitializedThis;
+				return (NULL != field) || !liveStack->uninitializedThis;
```

Comment:
@ChengJin01 the result of `findFieldFromCurrentRomClass ` is also used here and is not looking for ACC_STRICT so creating `isStrictField ` will not work in all cases.

Reply:
We can just leave `9ROMFieldShape *field = findFieldFromCurrentRomClass(romClass, fieldRef);` there.  e.g.
```
isFieldAccessCompatible(J9BytecodeVerificationData *verifyData, J9ROMFieldRef *fieldRef, UDATA bytecode, UDATA receiver, IDATA *reasonCode)
{
...
J9ROMFieldShape *field = findFieldFromCurrentRomClass(romClass, fieldRef); <-----
IDATA isStrictField = (NULL != field ) && J9_ARE_ALL_BITS_SET(field->modifiers, J9AccStrict); 
```
So we are simplify the rest of the `if` statements with a clear code except the place you mentioned above (where `field` is still used).

Reply:
Works for me. I updated the commit.